src/listener.ts

```diff
@@ -91,6 +92,10 @@ export class TCPListener extends EventEmitter<ListenerEvents> implements Listene
       }
     }
 
+    if (context.backlog != null && context.backlog <= 0) {
+      throw Error('backlog must be > 0')
+    }
```

Comment:
Since we just pass this on to  [server.listen](https://nodejs.org/dist/latest-v18.x/docs/api/net.html#serverlisten) is it necessary to validate this?